udata/core/dataset/search.py

```diff
@@ -82,9 +82,29 @@ def serialize(cls, dataset):
             'resources_count': len(dataset.resources),
             'organization': organization,
             'owner': str(owner.id) if owner else None,
-            'format': [r.format.lower() for r in dataset.resources if r.format]
+            'format': [r.format.lower() for r in dataset.resources if r.format],
+            'extras': dataset.extras
         }
 
+        serialized_resources = []
+        for resource in dataset.resources:
+            serialized_resources.append({
+                'id': resource.id,
+                'url': resource.url,
+                'format': resource.format,
+                'title': resource.title,
+                'schema': resource.schema,
```

Comment:
We'll need to read this information in the consumer. We could also put it at the dataset level, like we deed for format. What do you think?